src/page-objects/administration/CustomField.ts

```diff
@@ -0,0 +1,37 @@
+import type { Page, Locator } from '@playwright/test';
+import type { PageObject } from '../../types/PageObject';
+
+export class CustomField implements PageObject {
+    public readonly addNewSetButton: Locator;
+    public readonly customFieldRows: Locator;
+
+    constructor(public readonly page: Page) {
+        this.addNewSetButton = page.getByRole('button', { name: 'Add new set' });
+        this.customFieldRows = page.getByRole('row');
```

Comment:
@frobel Is this the only way to use a locator like this one?

Reply:
I want to provide with that solution the community to get als Rows via an property so that you can count or dig deeper with that locator.

Reply:
to actually work with the row elements it is better to use the method: `getLineItemByCustomFieldName()` (see below)